tasks/ports/freetds.rb

```diff
@@ -23,7 +23,7 @@ def configure_defaults
       end
 
       if windows?
-        opts << '--sysconfdir=C:\Sites'
+        opts << '--sysconfdir=C:/Sites'
```

Comment:
`C:/Sites` is a native Windows path. All windows versions accept forward slash as path separator (try `dir "C:/Sites"`). I generally avoid backslash as path separator even on windows, because it is used for escaping as well. See also: https://github.com/oneclick/rubyinstaller2/wiki/For-gem-developers#path-separator

In this particular case I got an error while freetds build about a missing directory `C:Sites`, where the backslash was obviously interpreted as escape character.